src/App.jsx

```diff
@@ -1,15 +1,22 @@
+import 'bootstrap'
```

Comment:
Does this actually do anything, as we're just linking the bootstrap css and js in the `index.html` anyway? Remove it and see if it still works.